packages/nimble-components/src/utilities/tests/storybook.ts

```diff
@@ -146,16 +146,19 @@ Overrides of properties are not recommended and are not theme-aware by default.
 ${howToOverride}
 </details>`;
 
-export const usageWarning = (componentName: string): string => `
+export const incubatingWarning = (
+    componentName: string,
+    statusLink: string
+): string => `
 <style class="code-hide">
-#usage-warning {
+#incubating-warning {
     color: red;
     font: var(${bodyFont.cssCustomProperty});
 }
 </style>
-<div id="usage-warning" class="code-hide">
-WARNING - The ${componentName} is still in development and considered
-experimental. It is not recommended for application use.
+<div id="incubating-warning" class="code-hide">
+WARNING - The ${componentName} is still incubating. It is not recommended for application use. 
+For more information on its status, <a href="${statusLink}">see here</a>.
```

Comment:
This makes the actual link text for screen readers, etc just "see here".
Can you make it so the link is more descriptive?

```suggestion
See the <a href="${statusLink}">incubating component status</a>.
```